.circleci/config.yml

```diff
@@ -701,7 +717,7 @@ jobs:
           path: device_logs.zip
           destination: device_logs.zip
 
-  Carthage release:
+  iOS Framework release:
     macos:
       xcode: "12.4.0"
```

Comment:
Can/should we update to 12.5?  Application Services just updated and Firefox-iOS is already on 12.5...

Reply:
Actually yeah, I thought I built the previous one on Xcode 12.4 for Focus, but actually it was 12.5 too